skimage/transform/_warps.py

```diff
@@ -898,3 +898,149 @@ def coord_map(*args):
     _clip_warp_output(image, warped, order, mode, cval, clip)
 
     return warped
+
+
+def _linear_polar_mapping(output_coords, k_ang, k_rad, center):
+    """Inverse mapping function to convert from cartesion to polar coordinates
+
+    Parameters
+    ----------
+    output_coords : ndarray
+        `(M, 2)` array of `(col, row)` coordinates in the output image
+    k_ang : float
+        Scaling factor that relates the intended number of rows in the output
+        image to angle: ``k_ang = nrows / (2 * np.pi)``
+    k_rad : float
+        Scaling factor that relates the radius of the circle bounding the
+        area to be transformed to the intended number of columns in the output
+        image: ``k_rad = ncols / radius``
+    center : tuple (row, col)
+        Coordinates that represent the center of the circle that bounds the
+        area to be transformed in an input image.
+
+    Returns
+    -------
+    coords : ndarray
+        `(M, 2)` array of `(col, row)` coordinates in the input image that
+        correspond to the `output_coords` given as input.
+    """
+    angle = output_coords[:, 1] / k_ang
+    rr = ((output_coords[:, 0] / k_rad) * np.sin(angle)) + center[0]
+    cc = ((output_coords[:, 0] / k_rad) * np.cos(angle)) + center[1]
+    coords = np.column_stack((cc, rr))
+    return coords
+
+
+def _log_polar_mapping(output_coords, k_ang, k_rad, center):
+    """Inverse mapping function to convert from cartesion to polar coordinates
+
+    Parameters
+    ----------
+    output_coords : ndarray
+        `(M, 2)` array of `(col, row)` coordinates in the output image
+    k_ang : float
+        Scaling factor that relates the intended number of rows in the output
+        image to angle: ``k_ang = nrows / (2 * np.pi)``
+    k_rad : float
+        Scaling factor that relates the radius of the circle bounding the
+        area to be transformed to the intended number of columns in the output
+        image: ``k_rad = width / np.log(radius)``
+    center : tuple (row, col)
+        Coordinates that represent the center of the circle that bounds the
+        area to be transformed in an input image.
+
+    Returns
+    -------
+    coords : ndarray
+        `(M, 2)` array of `(col, row)` coordinates in the input image that
+        correspond to the `output_coords` given as input.
+    """
+    angle = output_coords[:, 1] / k_ang
+    rr = ((np.exp(output_coords[:, 0] / k_rad)) * np.sin(angle)) + center[0]
+    cc = ((np.exp(output_coords[:, 0] / k_rad)) * np.cos(angle)) + center[1]
+    coords = np.column_stack((cc, rr))
+    return coords
+
+
+def warp_polar(image, center=None, radius=None, output_shape=None,
```

Comment:
```suggestion
def warp_polar(image, center=None, *, radius=None, output_shape=None,
```

This makes any arguments after `center` keyword-only.